eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -212,8 +242,9 @@ function Upload-Blobs
         & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true
     }
 }
-
-
+# VERIFY PACKAGES
+$apiUrl = "https://api.github.com/repos/$repoId"
+$pkgList = VerifyPackages -pkgRepository $packageRepository -artifactLocation $artifactLocation -workingDirectory $workingDirectory -apiUrl $apiUrl -releaseSha $releaseSha -continueOnError $continueOnError
```

Comment:
From this you should be able to pull out the tag and pass it explicitly. You shouldn't need to recreated it.

Reply:
I have a check on whether it is null or empty. When $pkgList return something invalid, we will manually create one.